docs-kits/kits/Circularity_KIT/page-adoption-view.md

```diff
@@ -553,94 +554,264 @@ The Product Passport aspect model could provide reliable details on the product
 
 ## Glossary
```

Comment:
Suggestion: please create a separate subpage under adoption view for the glossary to not deter from the core function of "adoption view"

Reply:
Done! Created dedicated Glossary Subpage